honeybee/radiance/analysisgrid.py

```diff
@@ -66,7 +66,7 @@ def from_json(cls, ag_json):
         """Create an analysis grid from json objects."""
         analysis_points = tuple(AnalysisPoint.from_json(pt)
                                 for pt in ag_json["analysis_points"])
-        return cls(analysis_points)
+        return cls(analysis_points = analysis_points, name = ag_json["name"], window_groups=None)
```

Comment:
no space after and before `=` when it's an argument name in a call. It should be: 
```python
cls(analysis_points=analysis_points, name=ag_json["name"], window_groups=None)
```